Makefile

```diff
@@ -58,9 +59,9 @@ test/unit:
 test/e2e:
 	$(CRI_BIN) run --rm \
 	           --volume $(CURDIR):$(CURDIR):Z \
-	           --volume $(HOME)/.kube:/root/.kube:Z \
+	           --volume $(subst %/,%,$(dir $(KUBECONFIG))):/root/.kube:Z \
```

Comment:
IIUC the new command is the equivalent of `dirname` [1].
Please consider using the output of dirname.

Please also consider mounting the directory as read-only [2].

[1] https://man7.org/linux/man-pages/man1/dirname.1.html
[2] https://docs.podman.io/en/latest/markdown/podman-run.1.html#volume-v-source-volume-host-dir-container-dir-options

Reply:
> IIUC the new command is the equivalent of dirname [1].
Please consider using the output of dirname.

not sure I understand - `dir` is not a linux cmd, it's GNU-make cmd.
`dirname` is not recongnized, notice the difference in output here:
```
           --volume ram-test-with-dir//home/ralavi/.kube/:ram:Z \
```
vs
```
           --volume ram-test-with-dirname/:ram:Z \
```

> Please also consider mounting the directory as read-only [2].

DONE

Reply:
This snippet works:
```suggestion
	           --volume $(shell dirname $(KUBECONFIG)):/root/.kube:Z \
```

Reply:
this will work, but it departs from the makeFile realm.
I prefer the former, but don't insist. 
changing.